packages/webb-ui-components/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,81 @@
+import * as AvatarPrimitive from '@radix-ui/react-avatar';
+import { Typography } from '@webb-dapp/webb-ui-components/typography';
+import cx from 'classnames';
+import React, { useMemo } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import { AvatarProps } from './types';
+
+/**
+ * Webb Avatar component
+ *
+ * Props:
+ *
+ * - `size`: Size of avatar - `md`: 24px, `lg`: 48px (default: `md`)
+ * - `darkMode`: Control darkMode using `js`, leave it's empty to control dark mode using `css`
+ * - `src`: Image source for avatar
+ * - `alt`: Alternative text if source is unavailable
+ * - `fallback`: Optional fallback text if source image is unavailable
+ * - `className`: Outer class name
+ *
+ * @example
+ *
+ * <Avatar alt="Webb Logo" src="webblogo.png" />
+ */
+export const Avatar: React.FC<AvatarProps> = (props) => {
+  const { alt, className: outerClassName, darkMode, fallback, size = 'md', src } = props;
+
+  const sizeClassName = useMemo(() => (size === 'md' ? 'w-6 h-6' : 'w-12 h-12'), [size]);
+
+  const classNames = useMemo(() => {
+    const borderColor =
+      typeof darkMode === 'boolean'
+        ? darkMode
+          ? ('border-mono-180' as const)
+          : ('border-mono-0' as const)
+        : ('border-mono-0 dark:border-mono-180' as const);
+
+    const bg =
+      typeof darkMode === 'boolean'
+        ? darkMode
+          ? ('bg-mono-180' as const)
+          : ('bg-mono-0' as const)
+        : ('bg-mono-0 dark:bg-mono-180' as const);
+
+    const text =
+      typeof darkMode === 'boolean'
+        ? darkMode
+          ? ('text-mono-0' as const)
+          : ('text-mono-180' as const)
+        : ('text-mono-180 dark:text-mono-0' as const);
+
+    return {
+      borderColor,
+      bg,
+      text,
+    };
+  }, [darkMode]);
+
+  const typoVariant = useMemo(() => (size === 'md' ? 'body4' : 'body1'), [size]);
```

Comment:
The parentheses here are redundant

Reply:
The parentheses are related to the code formatter config we are using (which is `prettier`). This line https://github.com/webb-tools/webb-dapp/blob/develop/.prettierrc.yml#L4